advanced_server.rb

```diff
@@ -0,0 +1,314 @@
+require 'sinatra'
+require 'octokit'
+require 'json'
+require 'openssl' # Used to verify the webhook signature
+require 'jwt'     # Used to authenticate a GitHub App
+require 'time'    # Used to get ISO 8601 representation of a Time object
+require 'logger'
+
+set :port, 3000
+
```

Comment:
I had to add this code in line 9 to get smee to receive events:
`set :bind, '0.0.0.0'`